setup.py

```diff
@@ -31,11 +36,40 @@ def include_dirs(self, include_dirs):
         self._include_dirs = include_dirs
 
 
+def get_liberfa_versions(path=os.path.join(LIBERFADIR, 'configure.ac')):
+    assert os.path.exists(LIBERFADIR)
```

Comment:
Not really important, but I'd remove this as I don't think the code can get here without it failing earlier, and even if it could, the open would fail, so there seems to be little benefit.

Reply:
+1